ble/GattClient.h

```diff
@@ -252,7 +261,11 @@ class GattClient {
      * @Note: Write commands (issued using writeWoResponse) don't generate a response.
      */
     void onDataWritten(WriteCallback_t callback) {
-        onDataWriteCallback = callback;
+        onDataWriteCallbackChain.add(callback);
+    }
+
+    WriteCallbackChain_t& onDataWritten() { 
```

Comment:
please add a documentation header. it might be good to add a code-example showing its use